pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dto/BlockTabularQuestionDto.java

```diff
@@ -29,10 +32,18 @@ public class BlockTabularQuestionDto {
     @ColumnName("block_guid")
     String blockGuid;
 
-    @ColumnName("shown_expr")
-    String shownExpr;
+    @Nested("sh_expr")
+    Expression shownExpression;
+
+    @Nested("en_expr")
+    Expression enabledExpression;
+
+    public String getShownExpr() {
+        return shownExpression.getText();
```

Comment:
Is it possible that `shownExpression` or `enabledExpression` reference will be set to NULL? If yes, I'd suggest modifying the getter expression to avoid NPE.

Reply:
Yup- it's not a required field- thanks for the catch!